includes/admin/class-amp-admin-pointers.php

```diff
@@ -96,7 +96,7 @@ private function get_pointers() {
 				array(
 					'selector'        => '#menu-posts-' . AMP_Story_Post_Type::POST_TYPE_SLUG,
 					'heading'         => __( 'AMP', 'amp' ),
-					'description'     => __( 'Head over here to create your first AMP story.', 'amp' ),
+					'description'     => __( 'Head over here to create your first story.', 'amp' ),
```

Comment:
Looks like you've removed `.` from some of the descriptions, should it be removed from here as well?

Reply:
Or I guess this is a different kind of description?

Reply:
This is the text within an admin pointer, which is quite a different area from the stories editor.